shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Serialisation/Generators/Bootstrap.hs

```diff
@@ -35,14 +32,5 @@ genSignature =
     <$> hedgehog (genBytes . fromIntegral $ DSIGN.sizeSigDSIGN ([] @a))
 
 genBootstrapAddress :: Gen (BootstrapAddress era)
-genBootstrapAddress = BootstrapAddress . snd <$> genByronVKeyAddr
+genBootstrapAddress = BootstrapAddress <$> hedgehog Byron.genAddress
```

Comment:
@redxaxder it looks like we were going out of our way to generate only non-redeemer bootstrap addresses. I suspect this is because we didn't intend redeemers to enter shelley. Generating redeem addresses for the trace generators would be a problem, but I think that these are never used in situations where we need to produce a signature that will validate on-chain. yea? I need to generate redeem addresses here so that my property test for `returnRedeemAddrsToReserves` isn't trivial.